### ISSUE TITLE
Capture.FULL not working with the latest Remote WebDriver

### ISSUE THREAD

**Issue by a-tikhomirov** [assertthat/selenium-shutterbug#96]
When using Selenide 6.2.0 (with Selenium 4.1.1 and WebDriver 4.1.1) and RemoteWebDriver Chrome 97

```java
Shutterbug.shootPage(WebDriverRunner.getWebDriver(), Capture.FULL)
```
causes:
```java
java.lang.IllegalArgumentException: object is not an instance of declaring class
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.assertthat.selenium_shutterbug.utils.web.Browser.defineCustomCommand(Browser.java:909)
	at com.assertthat.selenium_shutterbug.utils.web.Browser.takeFullPageScreenshotChromeCommand(Browser.java:715)
	at com.assertthat.selenium_shutterbug.utils.web.Browser.takeFullPageScreenshot(Browser.java:157)
	at com.assertthat.selenium_shutterbug.core.Shutterbug.shootPage(Shutterbug.java:154)
	at com.assertthat.selenium_shutterbug.core.Shutterbug.shootPage(Shutterbug.java:83)
	at com.assertthat.selenium_shutterbug.core.Shutterbug.shootPage(Shutterbug.java:69)
```

**Comment by anna-shchurok** [comment]
Checked on Selenoid `1.10.7` for Chrome `97.0` and Chrome `98.0`
using command `Shutterbug.shootPage(driver, Capture.FULL).save();`
didn't not manage to reproduce.

@a-tikhomirov is `97.0` version where the issue started to reproduce? on `96.0` it is all ok? 

Also `java.lang.IllegalArgumentException: object is not an instance of declaring class` usually means the type of driver is incompatible. What type of driver is actually passed? is it RemoteWebDriver?

**Comment by therealdjryan** [comment]
I get the error with Chrome 97 using RemoteWebDriver.

**Comment by gregdymek** [comment]
same here
`      java.lang.IllegalArgumentException: object is not an instance of declaring class
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:564)
        at com.assertthat.selenium_shutterbug.utils.web.Browser.defineCustomCommand(Browser.java:909)
        at com.assertthat.selenium_shutterbug.utils.web.Browser.takeFullPageScreenshotChromeCommand(Browser.java:715)
        at com.assertthat.selenium_shutterbug.utils.web.Browser.takeFullPageElementScreenshot(Browser.java:181)
        at com.assertthat.selenium_shutterbug.utils.web.Browser.takeFullElementScreenshotScroll(Browser.java:410)
        at com.assertthat.selenium_shutterbug.core.Shutterbug.shootElement(Shutterbug.java:292)
        at com.assertthat.selenium_shutterbug.core.Shutterbug.shootElement(Shutterbug.java:214)`

**Comment by gregdymek** [comment]
<img width="1201" alt="Screenshot 2022-02-20 at 12 39 28" src="https://user-images.githubusercontent.com/16454941/154840754-ab9e1029-b94f-4bdf-855e-e857212744a3.png">
when I called in for a delegate, there was no error (it just returned null), I'm using remote driver + selenium grid

**Comment by gregdymek** [comment]
Here is the fix:
`private void defineCustomCommand(String name, CommandInfo info) {
        try {
            Method defineCommand = HttpCommandExecutor.class.getDeclaredMethod("defineCommand", String.class, CommandInfo.class);
            defineCommand.setAccessible(true);
            CommandExecutor commandExecutor = ((RemoteWebDriver) this.driver).getCommandExecutor();
            if (commandExecutor instanceof TracedCommandExecutor) {
                Field delegateField = TracedCommandExecutor.class.getDeclaredField("delegate");
                delegateField.setAccessible(true);
                commandExecutor = (CommandExecutor) delegateField.get(commandExecutor);
            }
            defineCommand.invoke(commandExecutor, name, info);
        } catch (InvocationTargetException | IllegalAccessException | NoSuchMethodException | NoSuchFieldException e) {
            throw new RuntimeException(e);
        }
    }`

**Comment by artemryazantsev** [comment]
Also facing this issue (Selenium 4 + Selenoid)

**Comment by erichimm** [comment]
I'm facing the same issue (selenium-java 4.1.2 + chrome 98)

**Comment by glib-briia** [comment]
@gregdymek thank you for suggesting the fix. The fix will be deployed with v1.6

**Comment by therealdjryan** [comment]
@gregdymek, @glib-briia, 
The timing of your response if crazy. I just put in a PR with @gregdymek's fix
https://github.com/assertthat/selenium-shutterbug/pull/102/files?diff=unified&w=1
